test/executable_test.dart

```diff
@@ -336,7 +348,7 @@ void main() {
       expect(result.stdout, contains('No fatal infractions found, valid is good to go!'));
     });
 
-    test('passes when there are unused known common binary packages', () async {
+    test('passes when dependencies not used in lib provide executables', () async {
```

Comment:
I think this should be:
```suggestion
    test('passes when dependencies not used provide executables', () async {
```